test/connectors/CustomsDeclareExportsMovementsConnectorSpec.scala

```diff
@@ -69,14 +74,124 @@ class CustomsDeclareExportsMovementsConnectorSpec extends ConnectorSpec {
           )
       )
 
-      val request = DisassociateDUCRRequest("pid", "eori", "ucr")
-      await(connector.submit(request))
+      val request = DisassociateDUCRRequest("provider-id", "eori", "ucr")
+      connector.submit(request).futureValue
 
       verify(
         postRequestedFor(urlEqualTo("/consolidation"))
-          .withRequestBody(equalTo("{\"providerId\":\"pid\",\"eori\":\"eori\",\"ucr\":\"ucr\",\"consolidationType\":\"DISASSOCIATE_DUCR\"}"))
+          .withRequestBody(equalTo("""{"providerId":"provider-id","eori":"eori","ucr":"ucr","consolidationType":"DISASSOCIATE_DUCR"}"""))
       )
     }
   }
 
+  "fetch all Submissions" should {
+
+    "send GET request to the backend" in {
+
+      val expectedSubmission = exampleSubmissionFrontendModel()
+      val submissionsJson =
+        s"""[
+           |  {
+           |    "uuid":"${expectedSubmission.uuid}",
+           |    "eori":"$validEori",
+           |    "conversationId":"$conversationId",
+           |    "ucrBlocks":[
+           |      {
+           |        "ucr":"$correctUcr",
+           |        "ucrType":"D"
+           |      }
+           |    ],
+           |    "actionType":"Arrival",
+           |    "requestTimestamp":"${expectedSubmission.requestTimestamp}"
+           |  }
+           |]""".stripMargin
+
+      stubFor(
+        get(s"$FetchAllSubmissionsEndpoint?providerId=$providerId")
+          .willReturn(aResponse().withStatus(OK).withBody(submissionsJson))
+      )
+
+      val response = connector.fetchAllSubmissions(providerId).futureValue
+
+      val expectedUrl = s"$FetchAllSubmissionsEndpoint?providerId=$providerId"
+      verify(getRequestedFor(urlEqualTo(expectedUrl)))
+
+      response mustBe Seq(expectedSubmission)
+    }
+  }
+
+  "fetch single Submission" should {
+
+    "send GET request to the backend" in {
+
+      val expectedSubmission = exampleSubmissionFrontendModel()
+      val submissionJson =
+        s"""
+           |  {
+           |    "uuid":"${expectedSubmission.uuid}",
+           |    "eori":"$validEori",
+           |    "conversationId":"$conversationId",
+           |    "ucrBlocks":[
+           |      {
+           |        "ucr":"$correctUcr",
+           |        "ucrType":"D"
+           |      }
+           |    ],
+           |    "actionType":"Arrival",
+           |    "requestTimestamp":"${expectedSubmission.requestTimestamp}"
+           |  }
+           |""".stripMargin
+
+      stubFor(
+        get(s"$FetchSingleSubmissionEndpoint/$conversationId?providerId=$providerId")
+          .willReturn(aResponse().withStatus(OK).withBody(submissionJson))
+      )
+
+      val response = connector.fetchSingleSubmission(conversationId, providerId).futureValue
+
+      val expectedUrl = s"$FetchSingleSubmissionEndpoint/$conversationId?providerId=$providerId"
+      verify(getRequestedFor(urlEqualTo(expectedUrl)))
```

Comment:
As mentioned before, think its better just to do 
```
verify(
    getRequestedFor(urlEqualTo("/submissions/321?providerId=123"))
)
```
That way we test the contract